ci-operator/step-registry/rosa/aws/sts/hypershift/cilium/rosa-aws-sts-hypershift-cilium-chain.yaml

```diff
@@ -0,0 +1,21 @@
+chain:
+  as: rosa-aws-sts-hypershift-cilium
```

Comment:
Do we need the chain or only workflow is enough?

Reply:
I see there are a lot of steps there, so just following your style to use a chain  here.